github/handlers/UserProfileHandler.ts

```diff
@@ -0,0 +1,52 @@
+import { IRead, IPersistence, IHttp, IModify } from "@rocket.chat/apps-engine/definition/accessors";
+import { IRoom } from "@rocket.chat/apps-engine/definition/rooms";
+import { SlashCommandContext } from "@rocket.chat/apps-engine/definition/slashcommands";
+import { UIKitInteractionContext } from "@rocket.chat/apps-engine/definition/uikit";
+import { GithubApp } from "../GithubApp";
+import { sendNotification } from "../lib/message";
+import { userProfileModal } from "../modals/UserProfileModal";
+import { getAccessTokenForUser } from "../persistance/auth";
+
+export async function handleUserProfileRequest(
+    read: IRead,
+    context: SlashCommandContext,
+    app: GithubApp,
+    persistence: IPersistence,
+    http: IHttp,
+    room: IRoom,
+    modify: IModify,
+    uikitcontext?: UIKitInteractionContext
+){
+    let access_token = await getAccessTokenForUser(
+        read,
+        context.getSender(),
+        app.oauth2Config
+    );
+    if (access_token && access_token.token){
```

Comment:
I have written similar statements in code but I think we can use `access_token?.token`  moving forward, makes it more readable.

Reply:
Sure Samad, Changed 🫡